File: UNIDAD 3/C#/U3_P10.cs
using System;
namespace U3_P10
{
    internal class U3_P10
    {
        static void Main(string[] args){
            int hT,hE,hM=40;
            double ph,sueldo,phE,phM;
            Console.Write("Ingresa las horas trbajadas (en formato 24hrs): ");
            hT = Convert.ToInt32(Console.ReadLine());
            Console.Write("Ingresa el pago por hora: $");
            ph = Convert.ToInt32(Console.ReadLine());
            hE = hT - 40;
            if (hT<=40)
            {
                sueldo = hT * ph;
                Console.WriteLine("El sueldo del trabajador a la semana por trabajar " + hT + "hrs es: $" + sueldo);

            } else if (hT > 40)
            {
                phE = (ph * 2) * hE; phM = ph * hM;
                sueldo = phE + phM; 
                Console.WriteLine("El sueldo del trabajador a la semana por trabajar " + hT + "hrs es: $" + sueldo);

            }


        }
    }
}

File: UNIDAD 3/JAVA/U3_P10.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p10;
import java.util.Scanner;
/**
 *
 * @author fatim
 */
public class U3_P10 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int hT, hE,hM=40;
        double ph, sueldo,phE,phM;
        System.out.println("Ingresa las horas trabajadas (en formato 24hrs): ");
        hT = sc.nextInt();
        System.out.print("Ingresa el pago por hora: $");
        ph = sc.nextInt();
         hE=hT-40;
       
        
        
        if(hT<=40){
            sueldo = hT*ph;
            System.out.println("El sueldo del trabajador a la semana por trabajar " + hT + " hrs es: $" + sueldo);
        }
        else if(hT > 40){
         phE = (ph * 2) * hE; phM=hM*ph;  
         sueldo = phE+phM;
         System.out.println("El sueldo del trabajador a la semana por trabajar " + hT + " hrs es: $" + sueldo);
        }
    }
}
